.github/dependabot.yml

```diff
@@ -4,23 +4,31 @@
 #
 version: 2
 updates:
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  target-branch: master
+  reviewers:
+  - giantswarm/team-firecracker
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  target-branch: master
+  reviewers:
+  - giantswarm/team-firecracker
 - package-ecosystem: gomod
   directory: "/"
   schedule:
-    interval: daily
+    interval: weekly
     time: "04:00"
   open-pull-requests-limit: 10
   reviewers:
-  - giantswarm/team-firecracker-engineers
+  - giantswarm/team-firecracker
```

Comment:
i dont thnik we want this change

Reply:
/cc @MarcelMue @kopiczko

Reply:
Comes from this value: https://github.com/giantswarm/github/blob/1773a9a7696c1783016c61a1f93cb3155d171b94/repositories/meta.yaml#L61

IMO could just be changed to the `-engineers` one

Reply:
See https://github.com/giantswarm/github/pull/42.